Sources/MapboxNavigation/Constants+InstructionsCard.swift

```diff
@@ -1,19 +0,0 @@
-import UIKit
-import CoreLocation
-
-/// :nodoc:
-struct InstructionsCardConstants {
```

Comment:
These properties are no longer used anywhere (except `highlightDistance`, which was moved to `Constants.swift` and renamed to `InstructionCardHighlightDistance`).